### ISSUE TITLE
Incorrect Gettext::exists logic

### ISSUE THREAD

**Issue by sergeyklay** [phalcon/cphalcon#11310]
I found a logical bug in the Gettext Adapter by conducting a small test.
The translation [db example](https://github.com/phalcon/cphalcon/tree/fbdea4a3cf7180b115d5f74cedbf59f279f47630/unit-tests/locale)

**How to reproduce.**

```php
        $t = new \Phalcon\Translate\Adapter\Gettext(
            [
                'locale'    => 'en_US.utf8',
                'directory' => 'path/to/translation/gettext'
            ]
        );
```

**Actual**
```php
       var_dump($t->exists('AAA-BBB-CCCCC')); // bool(true)
```

**Expected**
```php
       var_dump($t->exists('AAA-BBB-CCCCC')); // bool(false)
```

**po-file**
```
# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# ZhuZongXin <dreamsxin@qq.com>, 2014.
#
msgid ""
msgstr ""
"Project-Id-Version: phalcon\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2014-05-17 13:45+0800\n"
"PO-Revision-Date: 2014-05-17 16:11+0800\n"
"Last-Translator: ZhuZongXin <dreamsxin@qq.com>\n"
"Language-Team: LANGUAGE <dreamsxin@qq.com>\n"
"Language: en\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Generator: Poedit 1.6.5\n"
"X-Poedit-SourceCharset: UTF-8\n"

msgid "你好！"
msgstr "Hello!"

msgid "你好 %name%！"
msgstr "Hello %name%!"
```

**Comment by Jurigag** [comment]
Well from php manual:

Returns a translated string if one is found in the translation table, or the submitted message if not found.

**Comment by sergeyklay** [comment]
Fixed in `2.0.x`
